src/codegen/compile.ml

```diff
@@ -1577,6 +1580,7 @@ module Tagged = struct
     | Null -> 27l
     | OneWordFiller -> 29l
     | FreeSpace -> 31l
+    | ArraySliceMinimum -> 32l
```

Comment:
I wonder if we need to extend `can_have_tag` although it appears to be unused to me (referenced once from an unused function).

Reply:
I think it's fine as is, since the tag shouldn't occur in generated code and the conservative approximation in can_have_tag should do the right thing.

Reply:
Maybe we should remove this unused function (in a separate PR) as it could be a pitfall for the future (if we further adjust the tag logic)?

Reply:
Yeah, I have not clue why it exists. I guess git blame might reveal who introduced it.

Reply:
I have put this on my backlog for future code refactoring (in a separate PR).